library/src/containers/Messages/Message.tsx

```diff
@@ -49,6 +49,16 @@ export const Message: React.FunctionComponent<Props> = ({
   const externalDocs = message.externalDocs();
   const showInfoList = contentType || externalDocs;
 
+  // For some reason the parser is returning the wrong message ID,
+  // when retrieving reply() messages that's the reason retrieving it this way
```

Comment:
smells like a parser bug that you should not workaround but check in parser

Reply:
Okay, I'll create an issue in the parser

Reply:
Did you create the issue in the parser? I'm trying to find it without luck.